### PR TITLE
Issue 82

### PR DESCRIPTION
**Pull request by wthompson40**
Created dynamic popup modal element per table element in cache details.  Going to create followup issues for making the table in popup modal the correct height and properly pass tag from cache details table to modal table to properly display the block associated with a given tag.

### CODE REVIEW

**Comment by jbonzo** [comment]
You have some conflicts you need to resolve. Run `git fetch && git rebase origin/refactor` and it'll stop when it runs into the conflicts. once you resolve all the files in red run `git add -u && git rebase --continue`

File: src/browser/cacheDetail.css [hunk 1]
```diff
@@ -21,4 +21,80 @@ tr:nth-child(even) {background-color: #f2f2f2}
...
+    animation-duration: 0.4s
+}
+
+/* Add Animation */
+@-webkit-keyframes animatetop {
+    from {top:-300px; opacity:0} 
+    to {top:0; opacity:1}
+}
+
+@keyframes animatetop {
```
Comment (jbonzo) [inline src/browser/cacheDetail.css:68]: just curious what this is

**Comment by jbonzo** [comment]
Also can you recreate this PR and base it off of `master`

**Comment by wthompson40** [comment]
I just rebased, should be good now

File: src/browser/cacheDetail.css (hunk 1 above)
Comment (wthompson40) [reply src/browser/cacheDetail.css:68]: Animations for how the modal element appears

**Pull request closed without merging**